compiler warnings seen after bullseye update (other than mongoose)

see ballle98/AqualinkD#58

If you compile AqualinkD on the new bullseye version of Raspberry OS you will see these compiler warnings:

```
OS: 11 (bullseye)
GLIBC: ldd (Debian GLIBC 2.31-13+rpt2+rpi1+deb11u2) 2.31

net_services.c: In function ‘action_web_request’:
net_services.c:1163:51: warning: field precision specifier ‘.*’ expects argument of type ‘int’, but argument 3 has type ‘size_t’ {aka ‘long unsigned int’} [-Wformat=]
 1163 |     sprintf(buf, "action_web_request() request '%.*s' took",http_msg->uri.len, http_msg->uri.p);
      |                                                 ~~^~        ~~~~~~~~~~~~~~~~~
      |                                                   |                      |
      |                                                   int                    size_t {aka long unsigned int}
onetouch.c: In function ‘set_macro_status.part.0’:
onetouch.c:562:5: warning: ‘strncpy’ output may be truncated copying 13 bytes from a string of length 16 [-Wstringop-truncation]
  562 |     strncpy(_macros[0].name, _menu[2], 13);
      |     ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
onetouch.c:566:5: warning: ‘strncpy’ output may be truncated copying 13 bytes from a string of length 16 [-Wstringop-truncation]
  566 |     strncpy(_macros[1].name, _menu[5], 13);
      |     ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
onetouch.c:570:5: warning: ‘strncpy’ output may be truncated copying 13 bytes from a string of length 16 [-Wstringop-truncation]
  570 |     strncpy(_macros[2].name, _menu[8], 13);
      |     ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
utils.c: In function ‘_LOG’:
utils.c:487:3: warning: ‘strncpy’ output truncated before terminating nul copying 6 bytes from a string of the same length [-Wstringop-truncation]
  487 |   strncpy(message, strLevel, strlen(strLevel));
      |   ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
utils.c:487:3: warning: ‘strncpy’ output truncated before terminating nul copying 6 bytes from a string of the same length [-Wstringop-truncation]
utils.c:487:3: warning: ‘strncpy’ output may be truncated copying between 5 and 8 bytes from a string of length 8 [-Wstringop-truncation]

```

--- a/aq_panel.c
+++ b/aq_panel.c
@@ -112,7 +112,7 @@ setPanel("RS-8 Combo");
 char _panelString[60];
 void setPanelString()
 {
-  sprintf(_panelString, "%s%s-%d %s%s%s",
+  snprintf(_panelString, sizeof(_panelString), "%s%s-%d %s%s%s",
       isRS_PANEL?"RS":"",
       isPDA_PANEL?"PDA":"", // No need for both of these, but for error validation leave it in.
       PANEL_SIZE(),

--- a/aq_programmer.c
+++ b/aq_programmer.c
@@ -2215,7 +2215,7 @@ void send_cmd(unsigned char cmd)
 void force_queue_delete()
 {
   if (_pgm_command != NUL)
-    LOG(PROG_LOG, LOG_INFO, "Reall bad coding, don't use this in release\n");
+    LOG(PROG_LOG, LOG_INFO, "Really bad coding, don't use this in release\n");
 
   _pgm_command = NUL;
 }

